src/Mvc/test/WebSites/BasicWebSite/Controllers/RazorComponentsController.cs

```diff
@@ -51,16 +47,16 @@ public class RazorComponentsController : Controller
                 }
             };
 
-        [HttpGet("/components/{staticPrerender=true}")]
-        [HttpGet("/components/routable/{staticPrerender=true}")]
-        public IActionResult Index(bool staticPrerender)
+        [HttpGet("/components/{**slug}")]
+        [HttpGet("/components/routable/{**slug}")]
+        public IActionResult Index()
```

Comment:
This was only working because of the extra route parameter - it was getting bound to `staticPrerender` route parameter, and then conversion to bool would fail and the test ignored it.